plugins/app-backend/migrations/20211229105307_init.js

```diff
@@ -25,12 +25,11 @@ exports.up = async function up(knex) {
     table.comment(
       'A cache of static assets that where previously deployed and may still be lazy-loaded by clients',
     );
+    table
+      .increments('id', { primaryKey: true })
+      .comment('Primary key to distinguish unique lines from each other');
```

Comment:
I take it this is a requirement for clustered MySQL? Would it be possible to add the key only for that? In particular we should be able to skip the whole SQLite block in the next migration with that right?

Reply:
MySQL and other clustered databases typically want tables to have a primary key, for replication/consistency purposes. It also seems like it would be good to keep the table schemas as close as possible across the different types of databases. Is there a reason you're concerned about adding a primary key in SQLite (or Postgres)?

Reply:
@freben I'd be curious to hear what you think about keeping the schemas consistent across the different databases.